lib/bundles/inspec-habitat/profile.rb

```diff
@@ -185,6 +187,12 @@ def create_run_hook
       File.write(run_hook_file, run_hook_contents)
     end
 
+    def create_settings_file
```

Comment:
This is a settings/config file that is specific to our habitat package and contains only the `sleep_time` environment variable we've created to govern how often InSpec should run. It has no affect on other Habitat settings.

As I described in the PR description, this is necessary so that Habitat Supervisor is controlling a config (vs. when I had it only in the run hook) so that a config value change at run-time will cause Habitat to restart the service. If it doesn't have a config file, it will think no restart is necessary.